framework/src/media/utils/MediaUtils.cpp

```diff
@@ -462,5 +462,66 @@ bool header_parsing(unsigned char *buffer, unsigned int bufferSize, audio_type_t
 	}
 	return true;
 }
+
+void write_little_endian(FILE *fp, unsigned int num, unsigned short byte)
+{
+	unsigned char buf;
+	while (byte--) {
+		buf = num & 0xff;
+		fwrite(&buf, sizeof(char), 1 ,fp);
+		num >>= 8;
+	}
+}
+
+bool write_wav_header(FILE *fp, unsigned int channel, unsigned int sampleRate, audio_format_type_t pcmFormat, unsigned int fileSize) {
+	/**
+	*wave header is
+	*Chunk ID 'RIFF' (4byte) / Chunk Size (4byte) / Fomat 'WAVE' (4byte) /
+	*Chunk ID 'fmt ' (4byte) / Chunk Size (4byte) / Audio Format (2byte) / NumChannels     (2byte) /
+	*sample Rate     (4byte) / Byte Rate  (4byte) / Block Align  (2byte) / Bits Per sample (2byte) /
+	*Chunk ID 'data' (4byte) / Chunk Size (4byte)
+	*/
+
+	unsigned int bitPerSample = 0;
+	unsigned int blockAlign = 0;
+	unsigned int byteRate = 0;
+
+	if (fseek(fp, 0, SEEK_SET) != 0) {
+			medvdbg("file seek failed error\n");
+			return false;
+	}
+
+	switch (pcmFormat) {
+	case AUDIO_FORMAT_TYPE_S8:
+		bitPerSample = 8;
+		break;
+	case AUDIO_FORMAT_TYPE_S16_LE:
+		bitPerSample = 16;
+		break;
+	case AUDIO_FORMAT_TYPE_S32_LE:
+		bitPerSample = 32;
+		break;
```

Comment:
AUDIO_FORMAT_TYPE_S16_LE can be used as a default, but let me hear your opinion.

Reply:
In the MediaTypes.h file, we made all three variables for audio_format_type_e available. If we do not need the other two, i will delete it.